RecoParticleFlow/PFClusterProducer/interface/InitialClusteringStepBase.h

```diff
@@ -88,8 +108,9 @@ class InitialClusteringStepBase {
   }
   unsigned _nSeeds, _nClustersFound; // basic performance information
   const std::unordered_map<std::string,int> _layerMap;
-  std::unordered_map<int,std::pair<double,double> > 
-    _thresholds;
+
+  typedef std::tuple<std::vector<int> ,std::vector<double> , std::vector<double> > _i3tuple;
```

Comment:
CMS style is to capitalize types .

Leading underscore should be avoided in general, unless there is a good reason

Reply:
it looks like here and everywhere the vectors are of the same size and in each case values from the row are more important than the full vector in each "tuple" element.
It would be more efficient to have "vector<tuple<int, double, double>>" and perhaps more intuitive to use a struct that has self-descriptive values for each element (better than get<2> )